#
# class CustomError(Exception):
#     """Custom Error"""
#     print(f"Error Maina")
#
# # num = int(input())
# # num = 5
# # if num < 10:
# #     raise CustomError
#
# try:
#     raise CustomError("This is a custom Error Message")
# except CustomError as error:
#     print(error)

